models/comments.model.js

```diff
@@ -18,4 +18,9 @@ exports.insertCommentById = (bodyObj, reviewId) => {
             .then((data)=>{
                     return data.rows[0]
             })
+}
+
+exports.removeComments = (commentId) => {
+    return db.query(`DELETE FROM comments WHERE comment_id = $1;`,[commentId])
+        .then(()=>{return})
```

Comment:
We'll need to add a 404 error handler here for a non-existent ID, to pass the test I've mentioned in my other comment 👍